src/Areas/Extension/Commands/AzqrCommand.cs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using System.Runtime.InteropServices;
+using AzureMcp.Areas.Extension.Options;
+using AzureMcp.Commands;
+using AzureMcp.Services.ProcessExecution;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Areas.Extension.Commands;
+
+public sealed class AzqrCommand(ILogger<AzqrCommand> logger, int processTimeoutSeconds = 300) : GlobalCommand<AzqrOptions>()
+{
+    private const string CommandTitle = "Azure Quick Review CLI Command";
+    private readonly ILogger<AzqrCommand> _logger = logger;
+    private readonly int _processTimeoutSeconds = processTimeoutSeconds;
+    private readonly Option<string> _subscriptionIdOption = ExtensionOptionDefinitions.Azqr.SubscriptionId;
+    private readonly Option<string> _resourceGroupNameOption = ExtensionOptionDefinitions.Azqr.ResourceGroupName;
+    private static string? _cachedAzqrPath;
+
+    public override string Name => "azqr";
+
+    public override string Description =>
+        """
+        Runs Azure Quick Review CLI (azqr) commands to generate compliance/security reports for Azure resources.
```

Comment:
I think it would help if the description had more hints about when to call it.  What prompts have you been using to trigger this?